MapleServer2/Types/Player.cs

```diff
@@ -432,7 +432,7 @@ public void TrophyUpdate(string type, int addAmount, string code = "", string ta
 
             if (trophy.GradeCondition.ConditionCodes[0].Contains('-'))
```

Comment:
Not one for this PR, but something to consider - this will need to be expanded upon to work for some trophies, such as some item collection trophies that require you to (for example) loot all of X items. These include multiple condition codes, one for each item.